hatchways/team-penguin
# Invitation dialog 18b

majority of styles were fixed. I'm still having challenges with the width sizing of the text inputs. they are a little too wide. I tried to restyle DialogContent but was having alignment issues.

I did not install the cors package but can do that part in an integration ticket.

------

to test:

* the application needs to be in authenticated state (currently default but may change after auth changes are merged)

* from the left pane, click the Invite Friends button (under Friends tab)

* email input should handle either single email or list of emails (comma separated). currently there is validation for empty email or invalid email values but the error is not displayed. the error validation is possibly too broad (failing on any invalid email in the list).

* the copy button should work (currently the url is hard coded)

* the submit button should not work (cors was added in a different ticket and also the BE requires refactoring) but an http request should be made (should expect to fail)

* the close button (upper right) should also work
